localstack/services/sqs/sqs_listener.py

```diff
@@ -332,6 +332,18 @@ def forward_request(self, method, path, data, headers):
 
             elif action == "CreateQueue":
                 req_data = self.fix_missing_tag_values(req_data)
+
+                def _is_fifo():
+                    for k, v in req_data.items():
+                        if v == "FifoQueue":
```

Comment:
Took me some time to understand the logic in here, ;) but then @giograno pointed out that `req_data` contains the flattened attributes like `"Attribute.1.Name": "FifoQueue", "Attribute.1.Value": "true"`. Ideally we'd be operating on the parsed `Attribute` list directly, but given that this fix should be temporary and we'll be switching to our implementation as the default provider soon, this solution looks good to me for now. 👍

Reply:
If I see this correctly, it only checks whether  the value "FifoQueue" exists in the attribute list. Which means that a creation with FifoQueue: False will fail. Granted, that is usually not the case because people only use it if they want to change the default value from False to True, but it is technically a valid call.

Reply:
You are correct, I had a similar issue as Waldemar, wrapping your head around the proper way should not be done quickly :)